temporal_capi/src/plain_date_time.rs

```diff
@@ -242,6 +242,74 @@ pub mod ffi {
                 .map_err(Into::into)
         }
 
+        pub fn equals(&self, other: &Self) -> bool {
+            self.0 == other.0
+        }
+
+        pub fn compare(one: &Self, two: &Self) -> i32 {
```

Comment:
suggestion: I believe there should be a simpler way to implement these.

The built-ins that have an `IsoDate` internally should have a method called `compare_iso` that does this work for you.

The exception to this I believe is the time oriented types, which should implement `PartialOrd` and `Ord`. In these cases, you can use `Ord`'s method `cmp` and cast `Ordering` into an `i8`. Although, `ZonedDateTime` implements a `compare_instant` method instead of an `Ord` implementation.